src/app/common/transactions/bitcoin/coinselect/local-coin-selection.ts

```diff
@@ -102,21 +103,27 @@ export function determineUtxosForSpend({ feeRate, recipients, utxos }: Determine
     new BigNumber(estimateTransactionSize().txVBytes).multipliedBy(feeRate).toNumber()
   );
 
+  const changeAmount =
+    BigInt(getUtxoTotal(neededUtxos).toString()) - BigInt(amount.amount.toNumber()) - BigInt(fee);
+
   const outputs: {
     value: bigint;
     address?: string;
-  }[] = [
-    ...recipients.map(({ address, amount }) => ({
-      value: BigInt(amount.amount.toNumber()),
-      address,
-    })),
-    {
-      value:
-        BigInt(getUtxoTotal(neededUtxos).toString()) -
-        BigInt(amount.amount.toNumber()) -
-        BigInt(fee),
-    },
-  ];
+  }[] =
+    changeAmount <= BTC_P2WPKH_DUST_AMOUNT
```

Comment:
Is it ok to leave the dust amount in the wallet? @fbwoolf

Reply:
@alter-eggo, thoughts on these changes? It looks like you've made the most recent changes to this code?